stdlib/public/Darwin/Foundation/String.swift

```diff
@@ -49,15 +102,82 @@ extension String : _ObjectiveCBridgeable {
     self._forceBridgeFromObjectiveC(x, result: &result)
     return result != nil
   }
+  
+  @_effects(releasenone)
+  @inline(__always)
+  private static func _bridgeToSmall(
+    _ source: NSString,
+    _ len: Int,
+    _ encoding: CFStringBuiltInEncodings) -> String? {
+    return String(unsafeUninitializedCapacity: 15) { (ptr, outCount) in
+      let converted = _getBytes(
+        source,
+        CFRange(location: 0, length: len),
+        encoding.rawValue,
+        UnsafeMutableRawPointer(ptr.baseAddress!).assumingMemoryBound(to:
+          UInt8.self),
+        15, //maxBufLen
+        &outCount
+      )
+      
+      return converted == len
+    }
+  }
+  
+  @_effects(releasenone)
+  @inline(__always)
+  private static func _bridgeTagged(_ source: NSString) -> String? {
+      if _isObjCTaggedPointer(source) {
+        return _bridgeToSmall(source, _length(source), CFStringBuiltInEncodings.ASCII)
+      }
+      return nil
+  }
+  
+  @_effects(releasenone)
+  private static func _unconditionallyBridgeFromObjectiveC_nonTagged(
+    _ source: NSString
+    ) -> String {
+    let len = _length(source)
+    let sourceClass:AnyClass = _getClass(source)
+    
+    //Only try regular CF-based NSStrings for now due to things like
+    //NSLocalizedString assoc objects and NSAttributedString content proxies
+    //Also only try Strings we believe will fit into a SmallString for now
+    //In the future we should do mutable __NSCFStrings of any length here.
+    if sourceClass == nscfClass,
+      len <= 15,
+      let eager = _bridgeToSmall(source, len, CFStringBuiltInEncodings.UTF8) {
```

Comment:
Formatting: open delimiter needs its own line for multi-line expressions like this